droid-api/src/main/java/uk/gov/nationalarchives/droid/internal/api/DroidAPI.java

```diff
@@ -322,17 +424,17 @@ private <T> List<ApiResult> getApiResults(IdentificationRequest<T> request) thro
         boolean fileExtensionMismatch = resultCollection.getExtensionMismatch();
 
         return resultCollection.getResults()
-                .stream().map(res -> createApiResult(res, extension, fileExtensionMismatch, request.getIdentifier().getUri()))
+                .stream().map(res -> createIdentificationResult(res, extension, fileExtensionMismatch, request.getIdentifier().getUri()))
                 .collect(Collectors.toList());
     }
 
-    private ApiResult createApiResult(IdentificationResult result, String extension, boolean extensionMismatch, URI uri) {
+    private IdentificationResult createIdentificationResult(uk.gov.nationalarchives.droid.core.interfaces.IdentificationResult result, String extension, boolean extensionMismatch, URI uri) {
```

Comment:
Can't you just use IdentificationResult or interfaces.IdentificationResult?

Reply:
Actually, I've changed my mind, I've renamed the record APIIdentificationResult so this doesn't need the package qualifier now.